[2021-11-22 13:18:57,900] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: ELT-python-snowflake.Extract manual__2021-11-22T13:18:56.595247+00:00 [queued]>
[2021-11-22 13:18:57,914] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: ELT-python-snowflake.Extract manual__2021-11-22T13:18:56.595247+00:00 [queued]>
[2021-11-22 13:18:57,915] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-22 13:18:57,915] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-22 13:18:57,917] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-22 13:18:57,932] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): Extract> on 2021-11-22 13:18:56.595247+00:00
[2021-11-22 13:18:57,941] {standard_task_runner.py:52} INFO - Started process 1655 to run task
[2021-11-22 13:18:57,947] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ELT-python-snowflake', 'Extract', 'manual__2021-11-22T13:18:56.595247+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/ELT-python-snowflake.py', '--cfg-path', '/tmp/tmpwt8yekol', '--error-file', '/tmp/tmpknur02pr']
[2021-11-22 13:18:57,949] {standard_task_runner.py:77} INFO - Job 66: Subtask Extract
[2021-11-22 13:18:58,005] {logging_mixin.py:109} INFO - Running <TaskInstance: ELT-python-snowflake.Extract manual__2021-11-22T13:18:56.595247+00:00 [running]> on host bd3f5d3b0fb3
[2021-11-22 13:18:58,074] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ELT-python-snowflake
AIRFLOW_CTX_TASK_ID=Extract
AIRFLOW_CTX_EXECUTION_DATE=2021-11-22T13:18:56.595247+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-22T13:18:56.595247+00:00
[2021-11-22 13:18:58,076] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-11-22 13:18:58,077] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/elt/user-orders/extract-load/mysql_to_s3.py']
[2021-11-22 13:18:58,093] {subprocess.py:85} INFO - Output:
[2021-11-22 13:19:02,706] {subprocess.py:89} INFO - WARNING: An illegal reflective access operation has occurred
[2021-11-22 13:19:02,707] {subprocess.py:89} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.1.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-11-22 13:19:02,708] {subprocess.py:89} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-11-22 13:19:02,710] {subprocess.py:89} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-11-22 13:19:02,711] {subprocess.py:89} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-11-22 13:19:03,849] {subprocess.py:89} INFO - 21/11/22 13:19:03 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-11-22 13:19:04,447] {subprocess.py:89} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-11-22 13:19:04,448] {subprocess.py:89} INFO - Setting default log level to "WARN".
[2021-11-22 13:19:04,449] {subprocess.py:89} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2021-11-22 13:19:19,240] {subprocess.py:89} INFO - [Stage 0:>                                                          (0 + 0) / 1][Stage 0:>                                                          (0 + 1) / 1][Stage 0:===========================================================(1 + 0) / 1]                                                                                /home/***/.local/lib/python3.7/site-packages/snowflake/connector/options.py:97: UserWarning: You have an incompatible version of 'pyarrow' installed (4.0.1), please install a version that adheres to: 'pyarrow<5.1.0,>=5.0.0; extra == "pandas"'
[2021-11-22 13:19:19,243] {subprocess.py:89} INFO -   _expected_pyarrow_version,
[2021-11-22 13:19:19,244] {subprocess.py:89} INFO - uploading entity: USERS
[2021-11-22 13:19:19,245] {subprocess.py:89} INFO - new:
[2021-11-22 13:19:19,246] {subprocess.py:89} INFO - +---+----+---+-------+----------+----------+
[2021-11-22 13:19:19,247] {subprocess.py:89} INFO - | id|name|age|address|created_at|updated_at|
[2021-11-22 13:19:19,247] {subprocess.py:89} INFO - +---+----+---+-------+----------+----------+
[2021-11-22 13:19:19,249] {subprocess.py:89} INFO - +---+----+---+-------+----------+----------+
[2021-11-22 13:19:19,250] {subprocess.py:89} INFO - 
[2021-11-22 13:19:19,251] {subprocess.py:89} INFO - uploading entity: ORDERS
[2021-11-22 13:19:19,253] {subprocess.py:89} INFO - new:
[2021-11-22 13:19:19,254] {subprocess.py:89} INFO - +---+-------+-----+------+----------+----------+
[2021-11-22 13:19:19,255] {subprocess.py:89} INFO - | id|id_user|spent|status|created_at|updated_at|
[2021-11-22 13:19:19,258] {subprocess.py:89} INFO - +---+-------+-----+------+----------+----------+
[2021-11-22 13:19:19,260] {subprocess.py:89} INFO - +---+-------+-----+------+----------+----------+
[2021-11-22 13:19:19,262] {subprocess.py:89} INFO - 
[2021-11-22 13:19:20,732] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-22 13:19:20,782] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=ELT-python-snowflake, task_id=Extract, execution_date=20211122T131856, start_date=20211122T131857, end_date=20211122T131920
[2021-11-22 13:19:20,869] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-22 13:19:20,928] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
